packages/php-wasm/compile/php/Dockerfile

```diff
@@ -2069,6 +2087,7 @@ RUN set -euxo pipefail; \
 	-s MIN_NODE_VERSION=200900 \
 	-s INITIAL_MEMORY=1024MB \
 	-s ALLOW_MEMORY_GROWTH=1 \
+	-s ASSERTIONS=0 \
```

Comment:
This was removed mistakenly by #2231. Without disabled ASSERTIONS, php-wasm/web builds seem to fail to load due to assertion errors.